packages/acot-preset-wcag/src/rules/focusable-has-indicator.ts

```diff
@@ -0,0 +1,68 @@
+import { createRule } from '@acot/core';
+
+const SELECTOR = [
+  'input',
+  'select',
+  'textarea',
+  'a[href]',
+  'button',
+  '[tabindex]',
+  'audio[controls]',
+  'video[controls]',
+  '[contenteditable]:not([contenteditable="false"])',
+  'details>summary:first-of-type',
+  'details',
+].join(',');
+
+type Options = {};
+
+export default createRule<Options>({
+  type: 'contextual',
+  selector: SELECTOR,
+  immutable: false,
+  meta: {
+    description: 'Focusable element has visible focus indicator',
+    tags: ['wcag21aa', '2.4.7 Focus visible'],
+    recommended: true,
+  },
```

Comment:
```suggestion
  immutable: true,
  meta: {
    tags: ['wcag21aa', '2.4.7 Focus visible'],
    recommended: true,
  },
```

- If you just want to change the focus, the rule is immutable.
- `meta.description` is obsolete.
    - My mistake left it in the scaffdog template :innocent:

Reply:
Removed `meta.description` https://github.com/acot-a11y/acot/commit/ef2f0810ff7a62db5021ccfdc798727186ff1529 :pray:

Reply:
> If you just want to change the focus, the rule is immutable.

I just thought there are case that the element was focused, they are sometimes irreversible change. but I think almost they are reversible change too. It seems to me We need to define immutability of the rules, doesn't it?

Reply:
Your point is correct :ok_hand:

No documentation yet, but the immutable rule restores the focus on the `body`.
Right, in this case, the immutable rule is fine :+1:

https://github.com/acot-a11y/acot/blob/ef2f0810ff7a62db5021ccfdc798727186ff1529/packages/core/src/tester.ts#L75-L80